### DISCUSSION TITLE
[Question] Camera Field of View(frustum) not visible in viewport when using FrankaCubeLift environment in IsaacLab

### DISCUSSION THREAD

**Discussion by right-chan** [isaac-sim/IsaacLab#1121] [Q&A]
Hi, I’m currently using the FrankaCubeLift environment from `omni.isaac.lab_tasks.manager_based.manipulation.lift` for my experiments.

To utilize a camera attached to the end-effector, I modified the `usd_path` in `FRANKA_PANDA_CFG` from:
`omniverse://localhost/NVIDIA/Assets/Isaac/4.2/Isaac/IsaacLab/Robots/FrankaEmika/panda_instanceable.usd`
to:
`omniverse://localhost/NVIDIA/Assets/Isaac/4.2/Isaac/Robots/Franka/franka_alt_fingers.usd`

I am able to receive images from the camera without issues.

However, when I launch IsaacSim using the Omniverse Launcher and select the camera type, I can see the camera's field of view (frustum) in the viewport. 

But when I run the same environment using IsaacLab, the frustum is not visible in the viewport even though the camera is functioning properly.

Is there a specific setting or configuration in IsaacLab that I need to enable to see the camera’s frustum in the viewport?

Any advice or suggestions would be appreciated!

| **Omniverse Launcher Viewport**<br>Field of view is visible | **IsaacLab Viewport**<br>Field of view is **not** visible|
|:-------------------------------:|:---------------------:|
| ![Omniverse Launcher](https://github.com/user-attachments/assets/a3429634-41fe-4ef0-97f5-a036b9bd0fe8) | ![IsaacLab](https://github.com/user-attachments/assets/62ee8897-1290-4a45-bb08-f3840fd7a9d2) |

**Comment by glvov-bdai** [comment]
You could also try recreating the frustum as a visualization marker https://isaac-sim.github.io/IsaacLab/source/how-to/draw_markers.html
